examples/EU DST example

```diff
@@ -0,0 +1,52 @@
+/* EU DST example */
+/* version 1: dd 23-10-2024 */
+
+#include <RTClib.h>   // https://github.com/adafruit/rtclib 
+RTC_DS3231 rtc; 
+//RTC_DS1307 rtc;
+
+#define USEDST  true // Use DST (true or false)
+
+DateTime now;
+
+DateTime dstclock(DateTime n) { // Return the given (DST adjusted) date and time according to DST settings (aespecially for date/time checking)
+  
+  DateTime b, e;
+  
+  b = DateTime(n.year(), 3, 31, 3, 0, 0); b = DateTime(n.year(), 3, 31 - b.dayOfTheWeek(), 1, 0, 0); // Last sunday in march 1:00
+  e = DateTime(n.year(), 10, 31, 3, 0, 0); e = DateTime(n.year(), 10, 31 - e.dayOfTheWeek(), 3, 0, 0); // Last sunday in October 3:00
+
+  if (USEDST && (n > b) && (n < e)) n = n + TimeSpan(0,1,0,0); // adjust standard time if within summertime
+  return n;
+}; 
+
+DateTime getclock() { // Retrieve the (DST adjusted) date and time
+  
+  DateTime n, b, e;
+  
+  n = rtc.now();
+  b = DateTime(n.year(), 3, 31, 3, 0, 0); b = DateTime(n.year(), 3, 31 - b.dayOfTheWeek(), 1, 0, 0); // Last sunday in march 1:00
+  e = DateTime(n.year(), 10, 31, 3, 0, 0); e = DateTime(n.year(), 10, 31 - e.dayOfTheWeek(), 3, 0, 0); // Last sunday in October 3:00
+
+  if (USEDST && (n > b) && (n < e)) n = n + TimeSpan(0,1,0,0); // adjust standard time if within summertime
+  return n;
```

Comment:
Avoid code duplication:
```suggestion
  return dstclock(rtc.now());
```

Reply:
It is not duplicated. For both the begin and the end date the last sunday is relevant. So the function should be called twice.

Reply:
What I mean is that `getclock()` could be reduced to this:

```c++
DateTime getclock() {
  return dstclock(rtc.now());
}
```